src/pyproxy.c

```diff
@@ -138,6 +138,10 @@ EM_JS(int, pyproxy_new, (int ptrobj), {
     return Module.hiwire_new_value(Module.PyProxies[ptrobj]);
   }
 
+  return -2; /* this means HW_UNDEFINED */
```

Comment:
Can't we `return HW_UNDEFINED` directly then?

Reply:
Hi, I tried it first, but it doesn't work. Seems that the JS-code parts here are not touched by CPP.